config/database.js

```diff
@@ -0,0 +1,15 @@
+module.exports = ({ env }) => ({
+  defaultConnection: 'default',
+  connections: {
+    default: {
+      connector: 'bookshelf',
+      settings: {
+        client: 'sqlite',
```

Comment:
can we replace this with SQL like (MySQL, PostgreSQL) or NoSQL services "database should be hosted/managed independently"

Reply:
@chinzou yes, check this up: https://www.c-sharpcorner.com/article/connecting-to-mysql-database-in-strapi/

Reply:
that would be way better, messages are too much to be handled with SQLite in production